static/app/views/organizationGroupDetails/actions/index.tsx

```diff
@@ -475,17 +469,20 @@ class Actions extends Component<Props, State> {
                   priority: 'danger',
                   label: t('Delete'),
                   hidden: !hasAccess,
-                  disabled: !getIssueCapability(group.issueCategory, 'delete').enabled,
-                  onAction: () => this.openDeleteModal(),
+                  disabled: !deleteCap.enabled,
+                  tooltip: deleteCap.disabledReason,
+                  onAction: deleteCap.enabled ? () => this.openDeleteModal() : undefined,
                 },
                 {
                   key: 'delete-and-discard',
                   priority: 'danger',
                   label: t('Delete and discard future events'),
                   hidden: !hasAccess,
-                  disabled: !getIssueCapability(group.issueCategory, 'deleteAndDiscard')
-                    .enabled,
-                  onAction: () => this.openDiscardModal(),
+                  disabled: !deleteDiscardCap.enabled,
+                  tooltip: deleteDiscardCap.disabledReason,
+                  onAction: deleteDiscardCap.enabled
```

Comment:
Gonna clean this up once @vuluongj20 fixes up the issues with disabled dropdown items still running the action